katago_server/startposes/models/startpos.py

```diff
@@ -48,9 +48,10 @@ class StartPos(Model):
     objects = StartPosQuerySet.as_manager()
 
     class Meta:
-        verbose_name = _("StartPosesNetwork")
-        verbose_name_plural = _("Networks")
+        verbose_name = _("StartPos")
+        verbose_name_plural = _("StartPoses")
         ordering = ["-created_at"]
+        db_table = "startposes_startpos"
```

Comment:
Why do we need this ?

Reply:
It makes absolutely sure that this db_table is the same one as used in the non-managed model.